autosklearn/evaluation/train_evaluator.py

```diff
@@ -909,7 +909,11 @@ def get_splitter(self, D):
                     raise ValueError('Unknown CrossValidator.')
                 ref_arg_dict = __baseCrossValidator_defaults__[class_name]
 
-                y = D.data['Y_train'].ravel()
+                y = D.data['Y_train']
+                if D.info['task'] in CLASSIFICATION_TASKS and \
```

Comment:
Shouldn't also the y value of a single-output regression task be raveled?

Reply:
Yes, indeed. The single-output regression should be reval as well.